### PR TITLE
added 3 govermental Websites

### PR DESCRIPTION
**Pull request by prajjwallive**

### CODE REVIEW

File: data/websites.json [hunk 1]
```diff
@@ -331,29 +331,44 @@
...
+      "description": "Official Website of Chitwan National Park"
+    },
+    {
+      "name": "Department of National Parks and WildLife Conservation",
+      "url": "https://dnpwc.gov.np/",
+      "description": "Official Website of Department of National Parks and WildLife Conservation"
     }
   ]
-}
+}
```
Comment (thenishantsapkota) [inline data/websites.json:374]: Could you add a new line at the end of the file please.

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

File: data/websites.json [hunk 2]
```diff
@@ -331,29 +331,45 @@
...
+      "name": "Chitwan National Park",
+      "url": "https://chitwannationalpark.gov.np/",
+      "description": "Official Website of Chitwan National Park"
+    },
+    {
+      "name": "Department of National Parks and WildLife Conservation",
+      "url": "https://dnpwc.gov.np/",
+      "description": "Official Website of Department of National Parks and WildLife Conservation"
     }
+
```
Comment (prajjwallive) [inline data/websites.json:373]: added new line

**Review by prajjwallive — COMMENTED** [review]
updated changes

**Comment by prajjwallive** [comment]
Changes have been made you can review it

**Review by thenishantsapkota — APPROVED** [review]

**Comment by thenishantsapkota** [comment]
Thank you for your contribution @prajjwallive :heart:

**Comment by prajjwallive** [comment]
Awesome, thank you so much!

**Pull request merged**